.circleci/config.yml

```diff
@@ -293,8 +296,11 @@ workflows:
                     ]
                   }
             - slack/notify:
+                # Temporarily sending alarms to test alarm sending setup.
```

Comment:
Isn't it the build that has failed semi-regularly, rather than the deploy / service update?

Reply:
Ah yes you're right. But we should alarm if either fail - and merging this will be enough to tell us whether generating an alarm like this works or not.

Reply:
Added notification for the build in https://github.com/thebiggive/donate-frontend/pull/1394/commits/5332e0b539fdb1b9572a0c8ad7d8201f3d456d69 . I don't know why Circle doesn't seem to have an option to add a post-step for the workflow as a whole, rather than just each individual job.